docs/setups/basicbotsetup.md

```diff
@@ -42,14 +42,22 @@ If you don't know what you are doing, you might wanna read everything
     !!! warning
         Please do not leave `your bot prefix` or `your token here` in the code. Please replace it with your bot prefix and your token.
 
+    !!! tip inline end
+        Stockfish path is optional. If you want to input it. Please choose stockfish as stockfish_20011801_32bit.exe. You can rename it to that
+
+        Ignore only takes in a list of commands you want to ignore. E.g. ignore=[Chess] Where Chess is imported like
+        Ignore only takes in a list of commands you want to ignore. E.g. `ignore=[Chess]` Where Chess is imported like
+
+        `from disgames.mixins.chess import Chess`
+
     ```python
     from discord.ext import commands
     from disgames import register_commands
     import discord
 
     client = commands.Bot(command_prefix='your bot prefix', intents=discord.Intents.all())
 
-    register_commands(client)
+    register_commands(client, ignore=[], stockfish_path=None)
```

Comment:
why `ignore` and `stockfish_path` kwargs? im pretty sure it dont matter if we have it or not...

Reply:
https://github.com/andrewthederp/Disgames/blob/71bb3c5831eee98a0e07819cd263a978c1dc7e2a/disgames/__init__.py#L8